Sending build context to Docker daemon  2.048kB
Step 1/2 : FROM oraclelinux:latest
 ---> 3ca6f4a2fc9e
Step 2/2 : RUN yum -y update     2>&1 && yum -y install    tar openssl crontabs socat curl  2>&1
 ---> Using cache
 ---> 13299ba988c3
Successfully built 13299ba988c3
Successfully tagged myoraclelinux:latest
Fri, 23 Nov 2018 18:20:17 UTC
Testing branch: master
[1;31;32mProgress: [0m[1;31;32m0[0m[1;31;32m/1/21[0m
==Running le_test_dependencies please wait
no crontab for root
no crontab for root
curl installed.[1;31;32m [PASS][0m
crontab installed.[1;31;32m [PASS][0m
openssl installed.[1;31;32m [PASS][0m
socat installed.[1;31;32m [PASS][0m
------------------------------------------
[1;31;32mProgress: [0m[1;31;32m0[0m[1;31;32m/2/21[0m
==Running le_test_install please wait
./acme.sh install[1;31;32m [PASS][0m
/root/.acme.sh/acme.sh exists.[1;31;32m [PASS][0m
_contains '19 0 * * * "/root/.acme.sh"/acme.sh --cron --home "/root/.acme.sh" > /dev/null' '0 \* \* \* "/root/.acme.sh"/acme.sh --cron --home "/root/.acme.sh" > /dev/null'[1;31;32m [PASS][0m
/root/.acme.sh/acme.sh uninstall  > /dev/null[1;31;32m [PASS][0m
------------------------------------------
[1;31;32mProgress: [0m[1;31;32m0[0m[1;31;32m/3/21[0m
==Running le_test_uninstall please wait
./acme.sh install[1;31;32m [PASS][0m
/root/.acme.sh/acme.sh uninstall[1;31;32m [PASS][0m
/root/.acme.sh/acme.sh no exists.[1;31;32m [PASS][0m
equals [1;31;32m [PASS][0m
------------------------------------------
[1;31;32mProgress: [0m[1;31;32m0[0m[1;31;32m/4/21[0m
==Running le_test_installtodir please wait
./acme.sh install[1;31;32m [PASS][0m
/root/myle/acme.sh exists.[1;31;32m [PASS][0m
_contains '21 0 * * * "/root/myle"/acme.sh --cron --home "/root/myle" > /dev/null' '0 \* \* \* "/root/myle"/acme.sh --cron --home "/root/myle" > /dev/null'[1;31;32m [PASS][0m
/root/myle/acme.sh uninstall[1;31;32m [PASS][0m
------------------------------------------
[1;31;32mProgress: [0m[1;31;32m0[0m[1;31;32m/5/21[0m
==Running le_test_uninstalltodir please wait
./acme.sh install[1;31;32m [PASS][0m
/root/myle/acme.sh uninstall[1;31;32m [PASS][0m
/root/myle/acme.sh no exists.[1;31;32m [PASS][0m
equals [1;31;32m [PASS][0m
------------------------------------------
[1;31;32mProgress: [0m[1;31;32m0[0m[1;31;32m/6/21[0m
==Running le_test_install_config_home please wait
./acme.sh --install --config-home /root/etc/acme[1;31;32m [PASS][0m
/root/.acme.sh/acme.sh exists.[1;31;32m [PASS][0m
_contains '22 0 * * * "/root/.acme.sh"/acme.sh --cron --home "/root/.acme.sh" --config-home "/root/etc/acme" > /dev/null' '0 \* \* \* "/root/.acme.sh"/acme.sh --cron --home "/root/.acme.sh" --config-home "/root/etc/acme" > /dev/null'[1;31;32m [PASS][0m
/root/etc/acme/account.conf exists.[1;31;32m [PASS][0m
/root/.acme.sh/account.conf no exists.[1;31;32m [PASS][0m
/root/.acme.sh/acme.sh --cron --config-home /root/etc/acme > /dev/null[1;31;32m [PASS][0m
/root/.acme.sh/acme.sh --uninstall --config-home /root/etc/acme > /dev/null[1;31;32m [PASS][0m
/root/etc/acme/account.conf exists.[1;31;32m [PASS][0m
/root/.acme.sh/account.conf no exists.[1;31;32m [PASS][0m
------------------------------------------
[1;31;32mProgress: [0m[1;31;32m0[0m[1;31;32m/7/21[0m
==Running le_test_standandalone_renew please wait
/root/.acme.sh/acme.sh issue no testdocker.acme.sh[1;31;40m [FAIL] [0m
[1;31;40mRun Failed
[0m[1;31;40mdeactivate failed
[0m[Fri Nov 23 18:20:25 UTC 2018] Using stage ACME_DIRECTORY: https://acme-staging.api.letsencrypt.org/directory
[Fri Nov 23 18:20:25 UTC 2018] Getting new-authz for domain='testdocker.acme.sh'
[Fri Nov 23 18:20:27 UTC 2018] It seems there is an error, but it's recovered now, please try again.
[Fri Nov 23 18:20:27 UTC 2018] If you see this message for a second time, please report bug: https://github.com/Neilpang/acme.sh
cat: /root/.acme.sh/ca/acme-staging.api.letsencrypt.org/ca.conf: No such file or directory
[Fri Nov 23 18:20:27 UTC 2018] new-authz error: {"type":"urn:acme:error:unauthorized","detail":"No registration exists matching provided key","status": 403}
[Fri Nov 23 18:20:27 UTC 2018] Can not get domain new authz token.
------------------------------------------
[1;31;32mProgress: [0m[1;31;40m1[0m[1;31;32m/8/21[0m
==Running le_test_standandalone_renew_v2 please wait
/root/.acme.sh/acme.sh --issue -d testdocker.acme.sh --standalone --certpath '/acmetest/certs/domain.cer' --keypath '/acmetest/certs/domain.key'  --capath '/acmetest/certs/ca.cer'  --reloadcmd 'echo this is reload'  --fullchainpath  '/acmetest/certs/full.cer'[1;31;40m [FAIL] [0m
[1;31;40mRun Failed
[0m[1;31;40mdeactivate failed
[0m[Fri Nov 23 18:20:29 UTC 2018] Using stage ACME_DIRECTORY: https://acme-staging.api.letsencrypt.org/directory
[Fri Nov 23 18:20:29 UTC 2018] Getting new-authz for domain='testdocker.acme.sh'
[Fri Nov 23 18:20:31 UTC 2018] It seems there is an error, but it's recovered now, please try again.
[Fri Nov 23 18:20:31 UTC 2018] If you see this message for a second time, please report bug: https://github.com/Neilpang/acme.sh
[Fri Nov 23 18:20:31 UTC 2018] new-authz error: {"type":"urn:acme:error:unauthorized","detail":"No registration exists matching provided key","status": 403}
[Fri Nov 23 18:20:31 UTC 2018] Can not get domain new authz token.
------------------------------------------
[1;31;32mProgress: [0m[1;31;40m2[0m[1;31;32m/9/21[0m
==Running le_test_standandalone_renew_localaddress_v2 please wait
/root/.acme.sh/acme.sh --issue -d testdocker.acme.sh --standalone --local-address 0.0.0.0 --certpath '/acmetest/certs/domain.cer' --keypath '/acmetest/certs/domain.key'  --capath '/acmetest/certs/ca.cer'  --reloadcmd 'echo this is reload'  --fullchainpath  '/acmetest/certs/full.cer'[1;31;40m [FAIL] [0m
[1;31;40mRun Failed
[0m[1;31;40mdeactivate failed
[0m[Fri Nov 23 18:20:33 UTC 2018] Using stage ACME_DIRECTORY: https://acme-staging.api.letsencrypt.org/directory
[Fri Nov 23 18:20:34 UTC 2018] Getting new-authz for domain='testdocker.acme.sh'
[Fri Nov 23 18:20:35 UTC 2018] It seems there is an error, but it's recovered now, please try again.
[Fri Nov 23 18:20:35 UTC 2018] If you see this message for a second time, please report bug: https://github.com/Neilpang/acme.sh
[Fri Nov 23 18:20:35 UTC 2018] new-authz error: {"type":"urn:acme:error:unauthorized","detail":"No registration exists matching provided key","status": 403}
[Fri Nov 23 18:20:35 UTC 2018] Can not get domain new authz token.
------------------------------------------
[1;31;32mProgress: [0m[1;31;40m3[0m[1;31;32m/10/21[0m
==Running le_test_standandalone_listen_v4_v2 please wait
/root/.acme.sh/acme.sh --issue -d testdocker.acme.sh --standalone --listen-v4 --cert-file '/acmetest/certs/domain.cer' --key-file '/acmetest/certs/domain.key'  --ca-file '/acmetest/certs/ca.cer'  --reloadcmd 'echo this is reload'  --fullchain-file  '/acmetest/certs/full.cer'[1;31;40m [FAIL] [0m
[1;31;40mRun Failed
[0m[1;31;40mdeactivate failed
[0m[Fri Nov 23 18:20:38 UTC 2018] Using stage ACME_DIRECTORY: https://acme-staging.api.letsencrypt.org/directory
[Fri Nov 23 18:20:38 UTC 2018] Getting new-authz for domain='testdocker.acme.sh'
[Fri Nov 23 18:20:39 UTC 2018] It seems there is an error, but it's recovered now, please try again.
[Fri Nov 23 18:20:39 UTC 2018] If you see this message for a second time, please report bug: https://github.com/Neilpang/acme.sh
[Fri Nov 23 18:20:39 UTC 2018] new-authz error: {"type":"urn:acme:error:unauthorized","detail":"No registration exists matching provided key","status": 403}
[Fri Nov 23 18:20:39 UTC 2018] Can not get domain new authz token.
------------------------------------------
[1;31;32mProgress: [0m[1;31;40m4[0m[1;31;32m/11/21[0m
==Running le_test_standandalone_listen_v6_v2 please wait
Skipped by TEST_IPV6
[1;31;40mdeactivate failed
[0m[Fri Nov 23 18:20:40 UTC 2018] Using stage ACME_DIRECTORY: https://acme-staging.api.letsencrypt.org/directory
[Fri Nov 23 18:20:40 UTC 2018] Getting new-authz for domain='testdocker.acme.sh'
[Fri Nov 23 18:20:42 UTC 2018] It seems there is an error, but it's recovered now, please try again.
[Fri Nov 23 18:20:42 UTC 2018] If you see this message for a second time, please report bug: https://github.com/Neilpang/acme.sh
[Fri Nov 23 18:20:42 UTC 2018] new-authz error: {"type":"urn:acme:error:unauthorized","detail":"No registration exists matching provided key","status": 403}
[Fri Nov 23 18:20:42 UTC 2018] Can not get domain new authz token.
------------------------------------------
[1;31;32mProgress: [0m[1;31;40m5[0m[1;31;32m/12/21[0m
==Running le_test_standandalone_deactivate_v2 please wait
/root/.acme.sh/acme.sh --issue -d testdocker.acme.sh --standalone --certpath '/acmetest/certs/domain.cer' --keypath '/acmetest/certs/domain.key'  --capath '/acmetest/certs/ca.cer'  --reloadcmd 'echo this is reload'  --fullchainpath  '/acmetest/certs/full.cer'[1;31;40m [FAIL] [0m
[1;31;40mRun Failed
[0m[1;31;40mdeactivate failed
[0m[Fri Nov 23 18:20:44 UTC 2018] Using stage ACME_DIRECTORY: https://acme-staging.api.letsencrypt.org/directory
[Fri Nov 23 18:20:45 UTC 2018] Getting new-authz for domain='testdocker.acme.sh'
[Fri Nov 23 18:20:46 UTC 2018] It seems there is an error, but it's recovered now, please try again.
[Fri Nov 23 18:20:46 UTC 2018] If you see this message for a second time, please report bug: https://github.com/Neilpang/acme.sh
[Fri Nov 23 18:20:46 UTC 2018] new-authz error: {"type":"urn:acme:error:unauthorized","detail":"No registration exists matching provided key","status": 403}
[Fri Nov 23 18:20:46 UTC 2018] Can not get domain new authz token.
------------------------------------------
[1;31;32mProgress: [0m[1;31;40m6[0m[1;31;32m/13/21[0m
==Running le_test_standandalone please wait
/root/.acme.sh/acme.sh issue no testdocker.acme.sh[1;31;40m [FAIL] [0m
[1;31;40mRun Failed
[0m[1;31;40mdeactivate failed
[0m[Fri Nov 23 18:20:48 UTC 2018] Using stage ACME_DIRECTORY: https://acme-staging.api.letsencrypt.org/directory
[Fri Nov 23 18:20:49 UTC 2018] Getting new-authz for domain='testdocker.acme.sh'
[Fri Nov 23 18:20:50 UTC 2018] It seems there is an error, but it's recovered now, please try again.
[Fri Nov 23 18:20:50 UTC 2018] If you see this message for a second time, please report bug: https://github.com/Neilpang/acme.sh
[Fri Nov 23 18:20:50 UTC 2018] new-authz error: {"type":"urn:acme:error:unauthorized","detail":"No registration exists matching provided key","status": 403}
[Fri Nov 23 18:20:50 UTC 2018] Can not get domain new authz token.
------------------------------------------
[1;31;32mProgress: [0m[1;31;40m7[0m[1;31;32m/14/21[0m
==Running le_test_standandalone_SAN please wait
/root/.acme.sh/acme.sh issue no "testdocker.acme.sh" "testdocker2.acme.sh"[1;31;40m [FAIL] [0m
[1;31;40mRun Failed
[0m[1;31;40mdeactivate failed
[0m[Fri Nov 23 18:20:53 UTC 2018] Using stage ACME_DIRECTORY: https://acme-staging.api.letsencrypt.org/directory
[Fri Nov 23 18:20:53 UTC 2018] Getting new-authz for domain='testdocker.acme.shtestdocker2.acme.sh'
[Fri Nov 23 18:20:55 UTC 2018] It seems there is an error, but it's recovered now, please try again.
[Fri Nov 23 18:20:55 UTC 2018] If you see this message for a second time, please report bug: https://github.com/Neilpang/acme.sh
[Fri Nov 23 18:20:55 UTC 2018] new-authz error: {"type":"urn:acme:error:unauthorized","detail":"No registration exists matching provided key","status": 403}
[Fri Nov 23 18:20:55 UTC 2018] Can not get domain new authz token.
------------------------------------------
[1;31;32mProgress: [0m[1;31;40m8[0m[1;31;32m/15/21[0m
==Running le_test_standandalone_ECDSA_256 please wait
/root/.acme.sh/acme.sh issue no testdocker.acme.sh no ec-256[1;31;40m [FAIL] [0m
[1;31;40mRun Failed
[0m[1;31;40mdeactivate failed
[0m[Fri Nov 23 18:20:57 UTC 2018] Using stage ACME_DIRECTORY: https://acme-staging.api.letsencrypt.org/directory
[Fri Nov 23 18:20:57 UTC 2018] Getting new-authz for domain='testdocker.acme.sh'
[Fri Nov 23 18:20:59 UTC 2018] It seems there is an error, but it's recovered now, please try again.
[Fri Nov 23 18:20:59 UTC 2018] If you see this message for a second time, please report bug: https://github.com/Neilpang/acme.sh
[Fri Nov 23 18:20:59 UTC 2018] new-authz error: {"type":"urn:acme:error:unauthorized","detail":"No registration exists matching provided key","status": 403}
[Fri Nov 23 18:20:59 UTC 2018] Can not get domain new authz token.
------------------------------------------
[1;31;32mProgress: [0m[1;31;40m9[0m[1;31;32m/16/21[0m
==Running le_test_standandalone_ECDSA_256_renew please wait
/root/.acme.sh/acme.sh issue no testdocker.acme.sh no ec-256[1;31;40m [FAIL] [0m
[1;31;40mRun Failed
[0m[1;31;40mdeactivate failed
[0m[Fri Nov 23 18:21:01 UTC 2018] Using stage ACME_DIRECTORY: https://acme-staging.api.letsencrypt.org/directory
[Fri Nov 23 18:21:02 UTC 2018] Getting new-authz for domain='testdocker.acme.sh'
[Fri Nov 23 18:21:04 UTC 2018] It seems there is an error, but it's recovered now, please try again.
[Fri Nov 23 18:21:04 UTC 2018] If you see this message for a second time, please report bug: https://github.com/Neilpang/acme.sh
[Fri Nov 23 18:21:04 UTC 2018] new-authz error: {"type":"urn:acme:error:unauthorized","detail":"No registration exists matching provided key","status": 403}
[Fri Nov 23 18:21:04 UTC 2018] Can not get domain new authz token.
------------------------------------------
[1;31;32mProgress: [0m[1;31;40m10[0m[1;31;32m/17/21[0m
==Running le_test_standandalone_ECDSA_256_SAN_renew please wait
/root/.acme.sh/acme.sh issue no "testdocker.acme.sh" "testdocker2.acme.sh" ec-256[1;31;40m [FAIL] [0m
[1;31;40mRun Failed
[0m[1;31;40mdeactivate failed
[0m[Fri Nov 23 18:21:06 UTC 2018] Using stage ACME_DIRECTORY: https://acme-staging.api.letsencrypt.org/directory
[Fri Nov 23 18:21:06 UTC 2018] Getting new-authz for domain='testdocker.acme.shtestdocker2.acme.sh'
[Fri Nov 23 18:21:08 UTC 2018] It seems there is an error, but it's recovered now, please try again.
[Fri Nov 23 18:21:08 UTC 2018] If you see this message for a second time, please report bug: https://github.com/Neilpang/acme.sh
[Fri Nov 23 18:21:08 UTC 2018] new-authz error: {"type":"urn:acme:error:unauthorized","detail":"No registration exists matching provided key","status": 403}
[Fri Nov 23 18:21:08 UTC 2018] Can not get domain new authz token.
------------------------------------------
[1;31;32mProgress: [0m[1;31;40m11[0m[1;31;32m/18/21[0m
==Running le_test_standandalone_ECDSA_256_SAN_renew_v2 please wait
/root/.acme.sh/acme.sh --issue -d "testdocker.acme.sh" -d "testdocker2.acme.sh" --standalone --keylength ec-256 --certpath '/acmetest/certs/domain.cer' --keypath '/acmetest/certs/domain.key'  --capath '/acmetest/certs/ca.cer'  --reloadcmd 'echo this is reload'  --fullchainpath  '/acmetest/certs/full.cer'[1;31;40m [FAIL] [0m
[1;31;40mRun Failed
[0m[1;31;40mdeactivate failed
[0m[Fri Nov 23 18:21:10 UTC 2018] Using stage ACME_DIRECTORY: https://acme-staging.api.letsencrypt.org/directory
[Fri Nov 23 18:21:11 UTC 2018] Getting new-authz for domain='testdocker.acme.shtestdocker2.acme.sh'
[Fri Nov 23 18:21:12 UTC 2018] It seems there is an error, but it's recovered now, please try again.
[Fri Nov 23 18:21:12 UTC 2018] If you see this message for a second time, please report bug: https://github.com/Neilpang/acme.sh
[Fri Nov 23 18:21:12 UTC 2018] new-authz error: {"type":"urn:acme:error:unauthorized","detail":"No registration exists matching provided key","status": 403}
[Fri Nov 23 18:21:12 UTC 2018] Can not get domain new authz token.
------------------------------------------
[1;31;32mProgress: [0m[1;31;40m12[0m[1;31;32m/19/21[0m
==Running le_test_standandalone_ECDSA_384 please wait
/root/.acme.sh/acme.sh issue no "testdocker.acme.sh" no ec-384[1;31;40m [FAIL] [0m
[1;31;40mRun Failed
[0m[1;31;40mdeactivate failed
[0m[Fri Nov 23 18:21:15 UTC 2018] Using stage ACME_DIRECTORY: https://acme-staging.api.letsencrypt.org/directory
[Fri Nov 23 18:21:15 UTC 2018] Getting new-authz for domain='testdocker.acme.sh'
[Fri Nov 23 18:21:16 UTC 2018] It seems there is an error, but it's recovered now, please try again.
[Fri Nov 23 18:21:16 UTC 2018] If you see this message for a second time, please report bug: https://github.com/Neilpang/acme.sh
[Fri Nov 23 18:21:16 UTC 2018] new-authz error: {"type":"urn:acme:error:unauthorized","detail":"No registration exists matching provided key","status": 403}
[Fri Nov 23 18:21:16 UTC 2018] Can not get domain new authz token.
------------------------------------------
[1;31;32mProgress: [0m[1;31;40m13[0m[1;31;32m/20/21[0m
==Running le_test_standandalone_renew_idn_v2 please wait
Skipped by TEST_IDN
[1;31;40mdeactivate failed
[0m[Fri Nov 23 18:21:17 UTC 2018] Using stage ACME_DIRECTORY: https://acme-staging.api.letsencrypt.org/directory
[Fri Nov 23 18:21:17 UTC 2018] Getting new-authz for domain='testdocker.acme.sh'
[Fri Nov 23 18:21:19 UTC 2018] It seems there is an error, but it's recovered now, please try again.
[Fri Nov 23 18:21:19 UTC 2018] If you see this message for a second time, please report bug: https://github.com/Neilpang/acme.sh
[Fri Nov 23 18:21:19 UTC 2018] new-authz error: {"type":"urn:acme:error:unauthorized","detail":"No registration exists matching provided key","status": 403}
[Fri Nov 23 18:21:19 UTC 2018] Can not get domain new authz token.
------------------------------------------
[1;31;32mProgress: [0m[1;31;40m14[0m[1;31;32m/21/21[0m
==Running le_test_dnsapi please wait
Skipped by TEST_DNS
[1;31;40mdeactivate failed
[0m[Fri Nov 23 18:21:19 UTC 2018] Using stage ACME_DIRECTORY: https://acme-staging.api.letsencrypt.org/directory
[Fri Nov 23 18:21:20 UTC 2018] Getting new-authz for domain='testdocker.acme.sh'
[Fri Nov 23 18:21:21 UTC 2018] It seems there is an error, but it's recovered now, please try again.
[Fri Nov 23 18:21:21 UTC 2018] If you see this message for a second time, please report bug: https://github.com/Neilpang/acme.sh
[Fri Nov 23 18:21:21 UTC 2018] new-authz error: {"type":"urn:acme:error:unauthorized","detail":"No registration exists matching provided key","status": 403}
[Fri Nov 23 18:21:21 UTC 2018] Can not get domain new authz token.
------------------------------------------
